### ISSUE TITLE
[DPG] Support encode for duration

### ISSUE THREAD

**Issue by lmazuel** [Azure/autorest.java#2106]
Blocked by https://github.com/Azure/typespec-azure/issues/2885

**Comment by weidongxu-microsoft** [comment]
same as https://github.com/Azure/autorest.java/issues/2089

**Comment by weidongxu-microsoft** [comment]
Wait for TCGC release, current lint error
```
Client format "seconds" is unknown. Known values are "unixtime", "iso8601", "rfc1123"TypeSpec(@azure-tools/typespec-client-generator-core/unknown-client-format)
```
on `on "@clientFormat("seconds")"`

**Comment by weidongxu-microsoft** [comment]
released in 0.7.0
